### PR TITLE
Removing information about saved queries

### PR DESCRIPTION
**Pull request by prog8**
We're replacing saved queries with saved views. In addition to that we already cut the connection between Alert Rules and Saved Queries. This means all information about saved queries should be removed or modified.

### CODE REVIEW

File: docs/alerts/creating-logs-alerts.md [hunk 1]
```diff
@@ -1,41 +1,30 @@
...
 
-In a Logs App, *saved queries* are used to save search queries that you want to reuse.
+In a Logs App you can create an Alert Rule which will trigger notifications when a condition is met.
 
 For example, let's say you used *include/exclude filters* to search for *Elasticsearch slowlogs warnings*.
 
 ![Elasticsearch slowlogs warnings search](../images/alerts/image_0.png)
 
-Clicking on the bell icon creates a new *saved query*, where filters are transformed into `Query string`.
+Clicking on the bell icon creates a new Alert Rule with query and all filters applied.
```
Comment (adnanrahic) [inline docs/alerts/creating-logs-alerts.md:10]: Clicking on the bell icon creates a new Alert Rule with query and all filters applied.
=>
Clicking on the bell icon creates a new Alert Rule with the applied query and filters.

File: docs/alerts/creating-logs-alerts.md [hunk 2]
```diff
@@ -1,41 +1,30 @@
...
 
 ![image alt text](../images/alerts/image_1.png)
 
-You can save this query as is and use it in future searches.
-
-![image alt text](../images/alerts/image_2.png)
-
-Clicking on the *alert view* icon <img role="presentation" width="26" height="26" src="/docs/images/alerts/image_27.png" style="display:inline"> opens the view where you can edit *saved queries*.
+Let's say we want to get notified if the number of *slowlog warnings* reaches 10 in any 10 minutes. Notice that *Chart Preview* shows the threshold line to help
+you visualize the threshold value in context.
```
Comment (adnanrahic) [inline docs/alerts/creating-logs-alerts.md:15]: you visualize the threshold value in context.
=>
you visualize the threshold value in this context.

File: docs/alerts/creating-logs-alerts.md [hunk 3]
```diff
@@ -1,41 +1,30 @@
...
 ![image alt text](../images/alerts/image_3.png)
 
-The bell icon here means that *saved query* has *alert rule* attached to it.
-
-Let's attach an *alert rule* to the *saved query* we've just created - click on `Edit`.  
-
-### Enable Alerts
-
-By turning on `Enable alert` toggle *saved query* is expanded into *alert rule*.  
+Besides simple log count you can choose any numeric field in *Metrics* section.
```
Comment (adnanrahic) [inline docs/alerts/creating-logs-alerts.md:19]: Besides simple log count you can choose any numeric field in *Metrics* section.
=>
Apart from a simple log count you can choose any numeric field in the *Metrics* section.

File: docs/logs/faq.md [hunk 4]
```diff
@@ -606,28 +606,23 @@ later.
 
 ### What are Logs App Alerts?
 
-Logs App alerts are based on saved searches that trigger alert
-notifications when these saved searches meet a certain threshold
+Logs App alerts are based on full text search queries and filters. They trigger alert
+notifications when these query and filters meet a certain threshold
```
Comment (adnanrahic) [inline docs/logs/faq.md:610]: notifications when these query and filters meet a certain threshold
=>
notifications when the query and filters meet a certain threshold

File: docs/logs/faq.md [hunk 5]
```diff
@@ -606,28 +606,23 @@ later.
...
 condition in your logs - e.g. less than N matches in 5 minutes, more
-than M matches in 3 minutes, etc., or when such saved searches detect
+than M matches in 3 minutes, etc., or when search query detect
```
Comment (adnanrahic) [inline docs/logs/faq.md:612]: than M matches in 3 minutes, etc., or when search query detect
=>
than M matches in 3 minutes, etc., or when search the query detects

File: docs/logs/faq.md [hunk 6]
```diff
@@ -606,28 +606,23 @@ later.
...
 *sudden changes* in the number/volume of matching logs - i.e.,
 anomalies.
 
 ### How to create Alerts in Logs App?
 
-Alerts are added by clicking Save Query icon located next to the Logs App input field. Enter query info and select option to enable alerts. Alerts panel will expand and allow you to customize alert specific to that particular search query.
+Alerts are added by clicking bell icon located next to the Logs App input field. Enter query info. Alerts panel
```
Comment (adnanrahic) [inline docs/logs/faq.md:618]: Alerts are added by clicking bell icon located next to the Logs App input field. Enter query info. Alerts panel
=>
Alerts are added by clicking the bell icon located next to the Logs App input field. Here you enter query info. The alerts panel

File: docs/logs/faq.md [hunk 7]
```diff
@@ -606,28 +606,23 @@ later.
...
+will expand and allow you to customize alert specific to that particular search query. [Read more](../alerts/creating-logs-alerts.md)
```
Comment (adnanrahic) [inline docs/logs/faq.md:619]: will expand and allow you to customize alert specific to that particular search query. [Read more](../alerts/creating-logs-alerts.md)
=>
will expand and allow you to customize the alert to that particular search query. [Read more](../alerts/creating-logs-alerts.md)

File: docs/logs/faq.md [hunk 8]
```diff
@@ -606,28 +606,23 @@ later.
...
 
-<img alt="Save Alert in Logs App" src="/docs/images/logs/save-alert-in-logging-app.gif" title="Save Alert in Logs App">
+### How to view Alert Rules?
 
-### How to view Alerts?
+Click on Alert Icon located on the left navigation bar and then select Alert Rules.
```
Comment (adnanrahic) [inline docs/logs/faq.md:623]: Click on Alert Icon located on the left navigation bar and then select Alert Rules.
=>
Click on the Alert Icon located in the left navigation and then select Alert Rules.

File: docs/logs/reports-and-components.md [hunk 9]
```diff
@@ -21,16 +21,8 @@ When your Logs App is loaded the **Save and Reset buttons will be hidden** since
...
-
-- **Saved Queries / Alert Rules** - Right next to the save query / alert icon you can see the saved queries and alert rules, this will open a modal dialogue window with **all your saved queries and alert rules** displayed in a table component that displays:
-
-	- _Tag Color_ that can be assigned to any logs query while it is being saved by user using colour picker componenet 
-	- _Query Name_ to further identify and describe logs query
-	- _Query String_ will be string you use in logs search input field
-	- _Owner_	
-	- _10m, 1h, 12h, and 1d_ shortcuts with histogram for fast observability of query patterns across common time intervals
-	- Saved _query actions_ dropdown menu used to add new, edit, clone, reset, and delete saved logs report
+- **Add Alert** - Every query can be use as **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data.
```
Comment (adnanrahic) [inline docs/logs/reports-and-components.md:24]: - **Add Alert** - Every query can be use as **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data.

=>

- **Add Alert** - Every query can be use as an **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data.

File: docs/logs/reports-and-components.md [hunk 10]
```diff
@@ -21,16 +21,8 @@ When your Logs App is loaded the **Save and Reset buttons will be hidden** since
...
+Each alert query is visible in Alert Rules list.
```
Comment (adnanrahic) [inline docs/logs/reports-and-components.md:25]: Each alert query is visible in Alert Rules list.
=>
Each alert query is visible in the Alert Rules list.

**Review by adnanrahic — CHANGES_REQUESTED** [review]

File: docs/logs/faq.md (hunk 5 above)
Comment (prog8) [reply docs/logs/faq.md:612]: I don't think we meant "search the query" it is "search query". Of course, detects vs detect makes sense

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**